bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx

```diff
@@ -167,6 +167,7 @@ class VideoList extends Component {
         >
           <VideoListItem
             user={user}
+            usersLength={users.length}
```

Comment:
You don't need to pass this prop to the `video-list-item`, it's just duplicating a check being done already in the `video-list` on line `157`